easybuild/easyconfigs/b/bx-python/bx-python-0.8.4-foss-2019a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'bx-python'
+version = '0.8.4'
+
+github_account = 'bxlab'
+homepage = 'https://github.com/%(github_account)s/%(name)s'
+description = """The bx-python project is a Python library and associated set of scripts to allow for rapid
+ implementation of genome scale analyses."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+dependencies = [
+    ('LZO', '2.10'),
+    ('SciPy-bundle', '2019.03'),
+]
+
+prebuildopts = "export CPATH=$EBROOTLZO/include/lzo:$CPATH && "
+
+use_pip = True
+
+exts_list = [
```

Comment:
Use
exts_default_options = {'source_urls': [PYPI_SOURCE]}
Then you can drop the source_urls from the respective extenstions.

Reply:
I'll take note of this for next easyconfigs